### PR TITLE
add version subcommand for frizbee

### LINKED ISSUE #6: Add command for getting frizbee version
Doing `frizbee version` should output the binary's version.

### PR DESCRIPTION
**Pull request by kranurag7**
Fixes #6

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -64,8 +64,6 @@ linters:
     - forbidigo
     - forcetypeassert
     - gci
-    - gochecknoglobals
```
Comment (kranurag7) [inline .golangci.yml:67]: I looked at the linter docs and found no way to have a config to allow certain global variables, and it was the same for other linter as well.

**Comment by JAORMX** [comment]
The test failure seems unrelated

**Comment by kranurag7** [comment]
```bash
$ uname -ms
Linux x86_64 
$ crane digest nginx:1.25.3
sha256:2bdc49f2f8ae8d8dc50ed00f2ee56d00385c6f8bc8a8b320d0a294d9e3b49026
$ go test
PASS
ok  	github.com/stacklok/frizbee/pkg/containers	2.738s
```

**Review by evankanderson — APPROVED** [review]
It's a pity that we need to pass in a tagged version ID by hand, but I can understand why that is.

**Pull request merged**